src/manager.rs

```diff
@@ -0,0 +1,194 @@
+use compile::ExecutableDevEnv;
+use parse::Parser;
+use run::{ Execution, ExecutionEvent, Error as RunError };
+use rusqlite;
+use serde_json::error::Error as SerdeJsonError;
+use std::collections::HashMap;
+use std::io::Error as IOError;
+use std::path::{ Path, PathBuf };
+use std::sync::mpsc::RecvError;
+use transformable_channels::mpsc::{ channel, ExtSender };
+
+
+/// A ScriptManager error.
+#[derive(Debug)]
+pub enum Error {
+    NoSuchScriptError,
+    ScriptAlreadyRunning,
```

Comment:
There is already a similar error in `RunError`. I believe that you could merge them.